Documentation/seccomp-guide.md

```diff
@@ -0,0 +1,375 @@
+exposed to applications.
+This provides a clearly defined mechanism to build sandboxed environments, where
+processes can run having access only to a specific reduced set of system calls.
+
+In the context of containers, seccomp filtering is useful for:
+
+* Restricting applications from invoking syscalls that can affect the host
+* Reducing kernel attack surface in case of security bugs
+
+For more details on how Linux seccomp filtering works, see
+[seccomp(2)](http://man7.org/linux/man-pages/man2/seccomp.2.html).
+
+## Predefined seccomp filters
+
+By default, rkt comes with a set of predefined filtering groups that can be
+used to quickly build sanboxed environments for containerized applications.
+These sets are just easier ways for referencing groups of syscalls covering
+a single functional area or kernel subsystem. They can be further combined to
+build more complex filters, either by blacklisting or by whitelisting specific
+system calls. To distinguish these predefined groups from real syscall names,
+wildcard labels are prefixed with a `@` symbols and are namespaced.
+
+`appc` specs define two groups:
+
+ * `@appc.io/all` represents the set of all available syscalls.
+ * `@appc.io/empty` represents the empty set.
+
+rkt provides two default groups for generic usage:
+
+ * `@rkt/default-blacklist` represents a broad-scope filter than can be used for generic blacklisting
+ * `@rkt/default-whitelist` represents a broad-scope filter than can be used for generic whitelisting
+
+For compatibility reasons, two groups are provided mirroring [default Docker
+profiles](https://github.com/docker/docker/blob/master/docs/security/seccomp.md):
+
+ * `@docker/default-blacklist`
+ * `@docker/default-whitelist`
+
+When using stage1 images with systemd >= v230, some 
+[predefined groups](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#SystemCallFilter=)
+are also available:
+
+ * `@systemd/clock` for syscalls manipulating the system clock
+ * `@systemd/default-whitelist` for a generic set of typically whitelisted syscalls
+ * `@systemd/mount` for filesystem mounting and unmounting
+ * `@systemd/network-io` for socket I/O operationgs
+ * `@systemd/obsolete` for unusual, obsolete or unimplemented syscalls
+ * `@systemd/privileged` for syscalls which need super-user syscalls
+ * `@systemd/process` for syscalls acting on process control, execution and namespacing
+ * `@systemd/raw-io` for raw I/O port access
+
+When no seccomp filtering is specified, by default rkt whitelists all the generic
+syscalls typically needed by applications for common operations. This is
+the same set defined by `@rkt/default-whitelist`.
+
+The default set is tailored to stop applications from performing a large
+variety of privileged actions, while not impacting their normal behavior.
+Operations which are typically not needed in containers and which may
+impact host state, eg. invoking `umount(2)`, are denied in this way.
```

Comment:
By unmounting all the inaccessible procfs/sysfs. But yes, also `mount()`ing would fit here. Do you have any specific case you would like to see here?